python/django/python-guestbook/kubernetes-manifests/guestbook-app.deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: python-guestbook-app
+  labels:
+    app: python-guestbook
+    tier: app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: python-guestbook
+      tier: app
+  template:
+    metadata:
+      labels:
+        app: python-guestbook
+        tier: app
+    spec:
+      initContainers:
+      # Wait for the database to get ready before starting the server, otherwise
+      # the server will crash
+      - name: check-db-ready
+        image: postgres
+        command: ['sh', '-c',
+          'until pg_isready -h python-guestbook-db.default.svc.cluster.local -p 5432;
```

Comment:
can you replace the port with environment variable holding the port number? I see you have the port in DATABASE_PORT

Reply:
Thanks very much for your feedback. I tried to replace "5432" with "$DATABASE_PORT", but looks like "containers" and "initContainers" are in different environments. Those environment variables are only available in "containers" but not in "initContainers". I tried to find whether there are any ways to define environment variables for both "containers" and "initContainers" but was not able to find any.

Reply:
OK,  let's leave it as 5432 then. Thanks for checking the options.